testing/trino-product-tests/src/main/java/io/trino/tests/iceberg/TestSparkCompatibility.java

```diff
@@ -368,10 +368,14 @@ public void testIdBasedFieldMapping()
         String sparkTableName = sparkTableName(baseTableName);
 
         onPresto().executeQuery(format(
-                "CREATE TABLE %s (_struct ROW(rename BIGINT, keep BIGINT, drop_and_add BIGINT), _partition BIGINT) "
+                "CREATE TABLE %s (_struct ROW(rename BIGINT, keep BIGINT, drop_and_add BIGINT, \"CaseSensitive\" BIGINT), _partition BIGINT) "
```

Comment:
Would be good to create a non-lowercase column with eg Spark, as it may be we lowercase everything (perhaps we do not, but test could not depend on it)

Reply:
yeah with delimiters it didn't lowercase, but makes sense, changed it to Spark.